languages/go/build.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+check_command() {
+  if ! command -v "$1" &>/dev/null; then
+    printf '%s\n' "$1 is required to build locally. Please install $2."
+    exit 1
+  fi
+}
+
+check_command cargo Rust
+check_command go Go
+check_command npm Node.js
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+GO_LIB_DIR="$REPO_ROOT/languages/go/internal/cinterface/lib"
+
+pushd "$REPO_ROOT" || exit 1
+printf '%s\n\n' "Cleaning old builds..."
+rm -f languages/go/example/example && rm -rf "$GO_LIB_DIR"
+
+printf '%s\n\n' "Building binaries..."
+cargo build
```

Comment:
Should we expose a way of building in release mode?

Reply:
Fair enough. I can get that added pretty easily.